native/SampleApps/SmartSyncExplorer/SmartSyncExplorer.xcodeproj/project.pbxproj

```diff
@@ -1478,7 +1479,8 @@
 				MTL_ENABLE_DEBUG_INFO = NO;
 				PRODUCT_BUNDLE_IDENTIFIER = com.salesforce.SmartSyncExplorerUITests;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				SWIFT_VERSION = 3.0;
+				SWIFT_SWIFT3_OBJC_INFERENCE = On;
```

Comment:
Should it be on or off?

Reply:
This was the `Xcode` recommended setting.